### PR TITLE
Run benchmarks with dead nodes

### PR DESCRIPTION
**Pull request by asonnino**
- Facility to run benchmarks with faulty (dead) nodes.

### CODE REVIEW

File: consensus/src/core.rs [hunk 1]
```diff
@@ -300,9 +300,7 @@ impl Core {
 
         // Check if we can commit the head of the 2-chain.
         // Note that we commit blocks only if we have all its ancestors.
-        let mut commit_rule = b0.round + 1 == b1.round;
-        commit_rule &= b1.round + 1 == block.round;
-        if commit_rule {
+        if b0.round + 1 == b1.round {
```
Comment (LefKok) [inline consensus/src/core.rs:303]: Are the QCs of b0 and b1 already verified?

**Review by LefKok — APPROVED** [review]
This does not yet have the reputation system right?

File: consensus/src/core.rs (hunk 1 above)
Comment (asonnino) [reply consensus/src/core.rs:303]: If by "verified" you mean that all signatures check, then yes.

Comment (asonnino) [reply consensus/src/core.rs:303]: Is that what you meant?

**Pull request merged**